third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -2204,8 +2182,8 @@ function CSS_getComputedStyle({ node }) {
 
     // TODO: add pseudoType support - https://linear.app/replay/issue/RUN-953
 
+    let styleInfo;
```

Comment:
Weird that this never bugged out. Are we never coming in here? 😶

Reply:
I think if these functions were "use strict" it would have, but without it we were writing to/reading from `window.styleInfo`